LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The license of Sphinx is not 3-clause BSD. This should be follow upstream's license.
https://github.com/sphinx-doc/sphinx/blob/master/LICENSE

Reply:
Thank you for your comment. I updated to the sphinx License.